### PR TITLE
Standardise Margin and Refactor CSS for Home page 

### PR DESCRIPTION
**Pull request by daltonfury42**
Due to so many people working on the page, the CSS and spacing was not unified.

1. Pulled out common things into one place, dis some minor cleanup.
2. Increased spacing between elements
3. Common margin for all elements

TODO: Footer to be fixed in follow up PR
TODO: Move out navbar to common

PS: I've been very lavish with spacing, because I think modern clean designs have a lot of space between UI elements. Check the preview and let me know if it's alright.

### CODE REVIEW

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 1]
```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import styles from '../../../styles/homePage.module.scss';
 
 export const BenefitsInfo = () => (
-  <div className={styles['benefits-info']}>
+  <div className={`${styles['section']} ${styles['benefits-info']}`}>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/StaticInfos.jsx:5]: moved all the common CSS in all the sections into a common one.

File: simplq/src/styles/homePage.module.scss [hunk 2]
```diff
@@ -51,31 +51,42 @@
...
     @include standard-header();
     font-family: $header-font;
     font-size: 2rem;
   }
-  .benefits-container {
+  & > .container {
+    margin: 4rem 0;
+    @media screen and (max-width: 550px) {
+      margin: 2rem;
+    }
```
Comment (daltonfury42) [inline simplq/src/styles/homePage.module.scss:68]: I've been very generous with paddings and margins, but reducing them on smaller screens.

**Comment by maaverik** [comment]
Screenshot from 2020-10-17 20-28-04
 
This part doesn't line up anymore. Is that okay? Everything else looks fine to me. @akashkbaburajan what do you think?

**Comment by daltonfury42** [comment]
I guess it's because you've kept smaller width

**Comment by daltonfury42** [comment]
This is happening because I have intentionally increased the size of the cards. I have a also replaced one SVG in a follow up  PR. 

If rest looks fine, we. can merge this, and later just reduce the size of each card if required... but I think bigger cards look better.. but that's all subjective

**Comment by maaverik** [comment]
Okay, that's fine with me.

**Review by maaverik — APPROVED** [review]

**Pull request merged**